src/data/levels.js

```diff
@@ -85,6 +85,10 @@ levels[34] = {
   unlocksShopItem: items.garlicSeed.id,
 }
 
+levels[35] = {
```

Comment:
I'd like to keep crop unlock levels two apart to keep the game challenging and increase the challenged a bit. Would you be okay with moving this to level 36?

```suggestion
  levels[36] = {
```